web/src/components/steps/noticeLawyerChild/reviewNLC/pdf/Form40Layout.vue

```diff
@@ -3,239 +3,235 @@
-                        </template>
-                        <template v-slot:cell(value)="data">
-                            <div style="font-size:7pt !important; color:#000;">{{data.value}}</div>                                           
-                        </template>
-                </b-table>                
+            <div style="float: left; width: 30%; border:1px solid #414142; height: 100pt; opacity: 0.3;">
+                <p style="display: block;margin-top: 85pt;margin-left: 40pt;">COURT STAMP</p>
             </div>
-        </div> 
-        
-<!-- <1> -->
-        <section>
-            <underline-form 
-                style="text-indent:2px;display:inline-block; font-size: 9pt;" 
-                textwidth="17rem" 
-                beforetext="I" 
-                hint="(full name of lawyer)" 
-                :italicHint="false" :text="lawyerInformation.lawyerName | getFullName"/>
-
-            <underline-form 
-                style="text-indent:2px;display:inline-block; font-size: 9pt;" 
-                textwidth="19rem" 
-                beforetext=", of" 
-                hint="(firm name, if applicable)" 
-                :italicHint="false" :text="lawyerInformation.firmName"/>
-            <div style="text-indent:5px;display:inline; font-size: 9pt;"> 
-                am the lawyer for the following children:
-            </div> 
-                
-            <b-table
-                :items="childDetails"
-                :fields="childTableFields"
-                class="mt-4"
-                small
-                bordered>                    
-                <template v-slot:cell()="data">
-                    <div style="font-size:11pt;color:#000">{{data.value}}</div>                                           
-                </template>
-                <template v-slot:head(dob)>
-                    Child’s Date of Birth <br><span style="font-size:6pt; font-weight:normal;">(mmm/dd/yyyy)</span>                            
-                </template>                       
-            </b-table>
-
-        </section>
-           
-        <div style="margin-top: 1rem;"></div>  
-
-<!-- <2> -->
-        <section>
-            <div style="text-indent:5px;display:inline; font-size: 9pt;"> 
-                I will be representing the child(ren) identified in section 1 on the following issue(s):
-            </div> 
-
-            <div style="margin-top: 0.5rem;"></div>
-
-            <i>Select all options that apply</i>
-            <div class="marginleft2p5vue" style="margin:0.25rem 0 0 1.5rem;">
-                <check-box 
-                    class="marginleft" 
-                    checkbox="" 
-                    inline="inline" 
-                    boxMargin="0" 
-                    style="display:inline;"
-                    :check="listOfIssues.includes('Parenting Arrangements')?'yes':''" text="parenting arrangements"/>
+            <div style="width: 35%; float:right; font-size: 8pt;">
```

Comment:
@JonTaylorBCGov2 There is FormHeader common component. May be better to use that to keep it consistent.

Reply:
I tried using that, but it wouldn't produce the header - I was not sure what I was doing incorrectly, so I abandoned it in favour of something that worked for me. I'd be happy to change it, but I may need some assistance in getting it to work.

Reply:
Are you talking about 'family-law-act-app\web\src\components\utils\PopulateForms\components\FormHeader.vue' ?

I can get it to work, but Its not generic enough. Im currently making it more generic right now so my PR can use it
https://github.com/bcgov/family-law-act-app/pull/697

Reply:
Disregard, I wasnt working with the latest version. Im not observing the same as Jon